### ISSUE TITLE
Brainstorming for further commands, options, features 

### ISSUE THREAD

**Issue by svrnm** [svrnm/demomonkey#13]
Add ideas for further commands, options, features to this ticket.

**Comment by svrnm** [comment]
From existing capabilities using TamperMonkey:
- AppD Flowmap Icon replacements
- AppD Flowmap Connection css class replacements
- Hide/Show elements by css selector

**Comment by konrness** [comment]
Auto-save of configurations

**Comment by svrnm** [comment]
@konrness - there is an "auto-save" feature you can turn on over settings ("Save configuration on line break"), but I assume you are looking for something that helps if you click from one config to another and all your stuff is lost

**Comment by konrness** [comment]
Correct, it is a typical use case for me where I am quickly switching between two different configurations copy/pasting or modifying values. The use-case would be that when auto-save is enabled, it would save changes whenever the active configuration is unloaded.

**Comment by svrnm** [comment]
@konrness - can you check if this is fixed with the latest dev release:

https://bit.ly/demomonkey-dev

If yes, I will try to release this within the next few days.
